### PR TITLE
Update outdated links to odh services.

### PR DESCRIPTION
**Pull request by HumairAK**
SSIA

### CODE REVIEW

**Comment by HumairAK** [comment]
@hemajv

File: docs/odh/README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Monitoring: [Link][20]
 
 UI: https://grafana-route-opf-monitoring.apps.zero.massopen.cloud
```
Comment (hemajv) [inline docs/odh/README.md:23]: we should update this to the new grafana link right?

File: docs/odh/README.md [hunk 2]
```diff
@@ -54,29 +54,15 @@ AlertManager UI: http://alert-manager-opf-monitoring.apps.zero.massopen.cloud/
...
-    UI: https://hue-opf-datacatalog.apps.zero.massopen.cloud/
-
-    Login Credentials: User/Pass is `generic_user`/`operatefirst`
-
-    > Note: Login via OCP is [WIP][15]
-
--   ##### [ThriftServer][11]
-
-    Spark Jobs UI: http://thriftserver-opf-datacatalog.apps.zero.massopen.cloud/jobs/
+UI: https://trino-route-opf-trino.apps.zero.massopen.cloud
```
Comment (hemajv) [inline docs/odh/README.md:59]: should this be https://trino-secure-opf-trino.apps.zero.massopen.cloud/?

**Comment by HumairAK** [comment]
@hemajv fixed

**Pull request merged**